modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -6,53 +6,236 @@
 
 == Prerequisites
+- OpenID Connect requires an external Identity Provider
+- OpenID Connect requires ownCloud Server to be connected with an https://doc.owncloud.com/server/latest/admin_manual/configuration/user/user_auth_ldap.html[LDAP/AD] (local DB users can't log in via OpenID Connect without further measures)
+- A distributed Memcache setup is required to operate this app - such as Redis or Memcached properly. Please follow ref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
+
+== Components
+- Component diagram (oC Server / LDAP / IdP / OIDC App / oC Client)
+
+== Setup and configuration
+
+Setting up ownCloud Server to work with OpenID Connect requires a couple of components to work together:
+- ownCloud Server set up with an https://doc.owncloud.com/server/latest/admin_manual/configuration/user/user_auth_ldap.html[LDAP/AD user backend and a regular user sync]
+- An external Identity Provider connected to the same LDAP/AD user backend and configured to work with the ownCloud components
+- The https://marketplace.owncloud.com/apps/openidconnect[OpenID Connect App] installed on ownCloud Server
+- Configuration for the OpenID Connect App in `config.php` on ownCloud Server
+- Service discovery for the ownCloud Clients
+
+The below sections will explain these areas and provide configuration examples using https://github.com/Kopano-dev/konnect[Kopano Konnect] as the external Identity Provider.
+
+For the configuration examples, let's assume we have
+- ownCloud Server available as `https://cloud.example.com`
+- Kopano Konnect available as `https://idp.example.com`
+
+=== Set up and configure the Identity Provider
+
+In order to get your Identity Provider running and ready to be used with ownCloud, you have to obtain Kopano Konnect and run it with a set of configuration values which can be provided as environment variables. See the below sections for details.
+
+Specifically, you have to
+- provide basic configuration
+- connect Kopano Konnect to LDAP/AD
+- set up a reverse proxy to expose the routes required to connect to Kopano Konnect. You'll find instructions in the https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-a-webserver-for-konnect[Kopano Konnect documentation].
+
+TIP: Kopano Konnect can be set up via Docker. Images are available on Docker Hub (`kopano/kopano_konnect`).
+TIP: For general information on Kopano Konnect, please refer to the https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-kopano-konnect[Kopano Documentation]. The configuration below is an example to make Kopano Konnect work with ownCloud Server.
+
+==== Basic configuration
+
+To run Kopano Konnect, some basic configuration is necessary. Please replace the below environment variables with the paths and values suited for your environment.
```

Comment:
Personally I'd not document how to setup kopano (or any other 3rd party component), because there is more then enough documentation upstream (more then we can write) and things change over time, which means that our information is outdated sooner or later.